documentation/bin/createDocu.sh

```diff
@@ -18,7 +18,7 @@ do
 done
 
 export CLASSPATH_FILE='target/cp.txt'
-mvn clean test dependency:build-classpath -Dmdep.outputFile=${CLASSPATH_FILE}
+mvn clean test dependency:build-classpath -Dmaven.test.failure.ignore=true  -Dmdep.outputFile=${CLASSPATH_FILE}
```

Comment:
Is it really a cool idea to ignore the results of the tests?

* I know that we are not interested here in the unit tests. They should run elsewhere.

* We can also ask :"why is the docu generated here?" since here we are interested in the integration tests. In case we come to an agreement that the tests are void here we can drop the docu generation here in total (in that case we should check if the docu is generated somewhere else before merging - otherwise we don't know if the PR destroys the docu-gen.

But having failing tests and fixing the issue by ignoring the test results seems bogus IMHO. I would like to suggest to understand and fix the root cause why our tests are failing.

Reply:
Yes. See https://github.com/SAP/jenkins-library/pull/1352 for fixing tests.

However, I think generating the docu should not be blocked by tests.

Reply:
But I am not sure if the success run of tests is a requirement for the docu generation as it is not working without test execution, so test failures might lead to a wrong documentation!?
Maybe someone having implemented the docu generation can judge better.

Reply:
> I think generating the docu should not be blocked by tests.

I believe having failing tests will affect the documentation. The tests are running here since we retrieve some kind of metadata which we use later on for creating a where-use list wrt plugins. In the docu there is a list of used plugins for the steps. This list I mean ...

In case we have failing tests that list will be incomplete, resulting in error-prone docu . That's actually bad.